vortex-array/src/array/chunked/mod.rs

```diff
@@ -143,13 +145,25 @@ impl ArrayValidity for ChunkedArray<'_> {
 
 impl EncodingCompression for ChunkedEncoding {}
 
+impl ScalarSubtractFn for ChunkedArray<'_> {
+    fn scalar_subtract(&self, to_subtract: &Scalar) -> VortexResult<OwnedArray> {
```

Comment:
I tried an implementation where I called subtract on each of the chunks and passed back a chunkedarray and benchmarked both, perf was nearly identical so I kept this implementation because it's nicer/simpler.